tasks/install.yml

```diff
@@ -2,5 +2,7 @@
 
 - include_tasks: update.yml
 - include_tasks: upgrade.yml
+- include_tasks: keys.yml
+- include_tasks: repositories.yml
```

Comment:
tasks in `repositories.yml` use `ansible.builtin.apt_repository` and `ansible.builtin.deb822_repository` but these modules needs python3-apt and python3-debian "on the host that executes this module" which are installed in the next task in `dependencies.yml`